pyccel/codegen/printing/ccode.py

```diff
@@ -794,7 +794,7 @@ def _print_Module(self, expr):
         self._current_module = expr.name
         body    = ''.join(self._print(i) for i in expr.body)
 
-        global_variables = ''.join([self._print(d) for d in expr.declarations])
+        global_variables = ''.join(self._print(d) for d in expr.declarations)
```

Comment:
I agree with this change but it is not relevant to this PR and is probably just going to create a conflict with the main repo so I would rather leave it as is

Reply:
[8fc652f](https://github.com/pyccel/pyccel-cuda/pull/32/commits/8fc652f06ecdce7fe22d9c89cd5b1369cb390858)